client/test/constants.ts

```diff
@@ -2,6 +2,7 @@ import { ClusterType } from '../src';
 
 export const CLUSTER = ClusterType.parse(process.env.CLUSTER || 'localnet');
 export const VALIDATOR_URL = CLUSTER.solanaUrl();
+export const DOCUMENT_SIZE = 1_000;
```

Comment:
I suggest moving this to src/lib/constants and renaming it DEFAULT_DOCUMENT_SIZE. Then it can be used in the register.ts instead of hard-coding 1_000